src/Pages/Help/About.elm

```diff
@@ -58,16 +62,116 @@ subscriptions model =
 
 
 
+-- MARKDOWN
+
+
+content : Html msg
+content =
+    Markdown.toHtml [ class "markdown" ] """
+
+# About Chorus
+                    
+
+Status Report
+---------
+**Chorus Version**                      19.x-2.4.8
```

Comment:
All these shouldn't be hardcoded. Some are from chorus, some need to be fetched from kodi. Have a look at the original implementation for e.g.

Reply:
Hello @razzeee, I apologize for the delay in response. I was having my rigorous end-semester evaluations and therefore I was busy preparing for the same.  
I didn't find any updated or good XML file decoder, so I converted the XML file to JSON and then fetched the data. I am facing 2 issues:- 
1) Http.get wants url. So I am pushing the file on a local server using the json-server package and then giving the absolute url in Http.get. I am trying to find a way to use the relative path in Http.get for which I have asked in Elm-Slack.
2) I was not able to track down from where they are fetching KODI Version data in chorus2. If you could direct me toward the file or server.

Reply:
you can't convert the addon.xml, kodi needs it. having to maintain a second json would be fine, but inconvenient, so best would probably be a build step. at that point, you could also just build a small elm file with only the (version) constants, that should allow to sidestep the fetch problem.

Reply:
Hello @razzeee,  Pls review the implementation.